Kitodo-API/src/main/java/org/kitodo/api/dataformat/service/DivServiceInterface.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.dataformat.service;
+
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Interface for a service that handles access to the {@code <mets:div>}
+ * element.
+ *
+ * <p>
+ * A div is a node of an outline tree, which divides the description of the
+ * digitized work according to the principle of a table of contents. Each div
+ * refers to an ordered list of one or more views (METS idiom: areas) that mimic
+ * the section of the digitized work described by this node when consumed in the
+ * given order. Each div can be assigned an amount of meta-data describing the
+ * described section of the digitized work in abstracted form.
+ */
+public interface DivServiceInterface {
+    /**
+     * Returns services to access the views (METS idiom: areas) that mimic the
+     * section of the digitized work associated with this div. Since this area
+     * is currently not fully developed, it can currently be assumed that the
+     * number of areas is always 1.
+     *
+     * @return the areas of this div
+     */
+    List<AreaServiceInterface> getAreas();
+
+    /**
+     * Returns services for access to the outline level that is a child of this
+     * outline level.
+     *
+     * @return services for access to the children of this div
+     */
+    List<DivServiceInterface> getChildren();
+
+    /**
+     * Returns the human-readable name of this outline element.
+     *
+     * @return the label
+     */
+    String getLabel();
+
+    /**
+     * Returns all meta-data that describe this outline level.
+     *
+     * @return the meta-data
+     */
+    Collection<MetaDataServiceInterface> getMetaData();
+
+    /**
+     * Returns a variant of the human-readable name of this outline element,
+     * which has been normalized so that it can be properly sorted by primitive
+     * sorting algorithms.
+     *
+     * @return the orderlabel
+     */
+    String getOrderlabel();
```

Comment:
OrderLabel?

Reply:
The METS attribute is uppercase ORDERLABEL. So I concluded to call the getter without camel case.